operations/mimir-mixin/dashboards/ruler-reads-resources.libsonnet

```
@@ -0,0 +1,53 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+local filename = 'mimir-reads-resources.json';
+
+(import 'dashboard-utils.libsonnet') {
+  [filename]:
+    ($.dashboard('Ruler reads resources') + { uid: std.md5(filename) })
+    .addClusterSelectorTemplates(false)
+    .addRow(
+      $.row('Query-frontend')
```

I would make it clear it's the query-frontend dedicated to ruler. Same comment applies to other rows.

```suggestion
      $.row('Query-frontend (dedicated to ruler)')
```